src/services/Storage.ts

```diff
@@ -17,3 +23,97 @@ export const createNote = async (
   };
   return newPostFeedItem;
 };
+
+export class Store implements StoreInterface {
+  put(targetPath: string, _content: Content): Promise<URL> {
+    return Promise.resolve(
+      new URL(`${process.env.REACT_APP_UPLOAD_HOST}/${targetPath}`)
+    );
+  }
+
+  async putStream(
+    targetPath: string,
+    doWriteToStream: WriteStreamCallback
+  ): Promise<URL> {
+    const ws = new ServerWriteStream(targetPath);
+    await doWriteToStream(ws);
+    return new URL(`${process.env.REACT_APP_UPLOAD_HOST}/${targetPath}`);
+  }
+}
+
+type WriteCallback = (error: Error | null | undefined) => void;
+type EndCallback = () => void;
+
+const isFunction = (o: any) => typeof o == "function";
+const isUint8Array = (o: any) =>
```

Comment:
These don't seem like they belong in this file. Maybe add a `utilities.ts` file somewhere?

Reply:
moved